src/middleware/authentication/middleware.ts

```diff
@@ -1,7 +1,68 @@
 import { NextFunction, Request, Response } from "express";
+import { SignJWT } from "jose";
+import axios from "axios";
+import crypto from "crypto";
 
 async function main(request: Request, response: Response, next: NextFunction) {
+  const authHeader = request.headers["authorization"];
+  const token = authHeader && authHeader.split(" ")[1];
+
+  if (token == null) return response.sendStatus(401);
+
+  const tokenData = await introspectToken(token);
+
+  const { username, exp } = tokenData;
+  if (request.params.formId !== username) {
+    response.status(403).json({ message: "Invalid token" });
+    return;
```

Comment:
This return statement can be moved to the previous line. This is something I have used and seen on the ExpressJS documentation. So it would become `return response.status(403).json({ message: "Invalid token" });`

Reply:
Im actually waiting for your Biome merge, I think there are few things Id like to adopt from a styling pov

Reply:
I can merge Biome but also wanted to see if @bryan-robitaille had something to say about it :)

Reply:
Go for it and give it a shot.  Currently my only pain with it is that I use the same VSCode settings for this repo and form-client.  So everytime I open up form-client it tells me I don't have Biome as part of my dependencies.

Reply:
@wmoussa-gc just merged Biome